CVLByExample/Types/Structs/BankAccounts/certora/specs/structs.spec

```diff
@@ -129,12 +129,6 @@ hook Sstore _customers[KEY address user].accounts.length uint256 newLength {
     numOfAccounts[user] = newLength;
 }
 
-/**
- An internal step check to verify that our ghost works as expected, it should mirror the number of accounts.
- Once the sload is defined, this invariant becomes a tautology  
```

Comment:
> Once the sload is defined, this invariant becomes a tautology  

Is sload now defined nowadays, so is this why we see the sanity failure for trivial post condition of the invariant?